.github/workflows/test.yaml

```diff
@@ -81,7 +81,6 @@ jobs:
           protoc --version
           touch net/lp_rpc.proto net/redeemer.proto
           make net/lp_rpc.pb.go net/redeemer.pb.go
-          git diff --exit-code
```

Comment:
Isn't this good check to have?

Reply:
I'm not sure this is a good check to have, and the step should probably be removed altogether, because:
* different *minor* versions of protoc generate different files, I think it's not a good idea to force developers to install specific protoc version, which may not be available from OS package manager
* specific protoc version is not pinned above in this job, when it's installed through apt, so the check may break - as already happened for me locally
* no relevant info in CONTRIBUTING.md
* the check of auto generated file is neither semantic nor syntactic
* all protocol functions should be covered by unit tests explicitly

Let me know what you think. Maybe @darkdarkdragon can provide any additional context on this too.